extensions/sockets/src/main/java/org/mule/extension/socket/internal/SocketUtils.java

```diff
@@ -32,29 +33,22 @@ private SocketUtils() {
 
   }
 
-  public static final String DEFAULT_ENCODING = "UTF-8";
   private static final String SOCKET_COULD_NOT_BE_CREATED = "%s Socket could not be created correctly";
+  public static final String WORK = "work";
 
   /**
    * UDP doesn't allow streaming and it always sends payload when dealing with a {@link Message}
    */
   public static byte[] getUdpAllowedByteArray(Object data, String encoding, ObjectSerializer objectSerializer)
       throws IOException {
-    return getByteArray(data, true, false, encoding, objectSerializer);
+    return getByteArray(data, false, encoding, objectSerializer);
   }
 
-  public static byte[] getByteArray(Object data, boolean payloadOnly, boolean streamingIsAllowed, String encoding,
+  public static byte[] getByteArray(Object data, boolean streamingIsAllowed, String encoding,
                                     ObjectSerializer objectSerializer)
       throws IOException {
     if (data instanceof InputStream && !streamingIsAllowed) {
       throw new IOException("Streaming is not allowed with this configuration");
-    } else if (data instanceof Message) {
```

Comment:
it's no longer possible to receive an entire message on the source so this is no longer needed.